### PR TITLE
Do not artificially limit count queries

### PR DESCRIPTION
**Pull request by fmizzell**

### CODE REVIEW

File: modules/custom/dkan_sql_endpoint/src/Helper/GetStringsFromStateMachineExecution.php [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+  /**
+   * Private.
+   */
+  private function isStates($input): bool {
+    if (!is_array($input)) {
+      return FALSE;
+    }
+
+    return TRUE;
+  }
```
Comment (thierrydallacroce) [inline modules/custom/dkan_sql_endpoint/src/Helper/GetStringsFromStateMachineExecution.php]: How is the above function different from:
`return is_array($input);`

File: modules/custom/dkan_sql_endpoint/src/Helper/GetStringsFromStateMachineExecution.php [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+   * Private.
+   */
+  private function containsFirstState(array $states): bool {
+
+    if (in_array(0, $states)) {
+      return TRUE;
+    }
+
+    return FALSE;
+  }
```
Comment (thierrydallacroce) [inline modules/custom/dkan_sql_endpoint/src/Helper/GetStringsFromStateMachineExecution.php]: Similar to above comment:
`return in_array(0, $states);`

File: modules/custom/dkan_sql_endpoint/src/Helper/GetStringsFromStateMachineExecution.php (hunk 1 above)
Comment (fmizzell) [reply modules/custom/dkan_sql_endpoint/src/Helper/GetStringsFromStateMachineExecution.php]: It isn't!!! will change to make it more concise.

File: modules/custom/dkan_sql_endpoint/src/Helper/GetStringsFromStateMachineExecution.php (hunk 2 above)
Comment (fmizzell) [reply modules/custom/dkan_sql_endpoint/src/Helper/GetStringsFromStateMachineExecution.php]: It isn't!!! will change to make it more concise.

**Pull request merged**